.github/workflows/test-sqlite.yml

```diff
@@ -0,0 +1,24 @@
+name: test-sqlite
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

Comment:
Consider maybe also
```yml
types: [opened, reopened, synchronize]
```

Reply:
what does that do? (it seems to re-run on pushing to a PR already)

Reply:
To be more specific:
```yml
on:
  push:
    branches:
      - main
  pull_request:
    types: [opened, reopened, synchronize]
```

(syntax mutatis mutandis)